examples/text_to_image/train_text_to_image.py

```diff
@@ -971,6 +1059,7 @@ def collate_fn(examples):
         pipeline.save_pretrained(args.output_dir)
 
         if args.push_to_hub:
+            save_model_card(args, repo_id, images, repo_folder=args.output_dir)
```

Comment:
Can you separately create the images that are saved to the model card here? Feel free to just pull the subset of code out of the `log_validation_images` that creates the images into a helper method.

Reply:
Either that, or skip the image block in the model card. I'd prefer to have the images, but we may not be able to generate them unless we make up a prompt that may not be ideal for the particular fine-tune the user performed.

Reply:
If you see the logic in the new `save_model_card()` function, that is how it's currently done. 

If there are validation images to save to the model card, `img_str` will be crafted accordingly, otherwise, it will be none.

Reply:
Yeah, that works for me.